src/cmd/usb-util.js

```diff
@@ -239,10 +239,6 @@ async function reopenDevice(device) {
 			device = await openDeviceById(id);
 			// check if we can communicate with the device
 			if (device.isOpen) {
-				// attempt to make a request to the device to check if it responds
-				if (!device.isInDfuMode) {
-					await device.getDeviceMode({ timeout: 1500 });
```

Comment:
In order to test this behavior: 
* Connect an electron without antenna and wait some seconds.
* Attempt to flash the device

** Behavior ** 
The device will stuck trying to get the device Mode and throw errors regarding `Memory allocation error`
The reason is the device is trying to connect to the cloud and can't handle this specific request (but can handle an listeningMode or dfuMode without issues)